index.html

```diff
@@ -68,7 +68,7 @@ <h2>Members and Local Organizations</h2>
 		{% for member in site.data.members %}
 			{% if member.status == "member" %}
 				<div class="small-gallery-item">
-					<img src="/assets/images/member_photos/{{ member.image }}" class="gallery-item-img"/>
```

Comment:
@Simonmoush For forks to publish, I think the move is to get into the habit of using relative URLs in these templates. For absolute URLs in data files there's the `relative_url` Liquid filter, which I used in a couple places to get beatboxchad.github.io/nm4p_website/ to work